docs/advanced-usage/clerk-idp.mdx

```diff
@@ -84,20 +85,63 @@ The endpoint returns detailed token information such as if the token is still ac
 > [!WARNING]
 > This endpoint is protected. You must provide your Clerk Client ID and Client Secret credentials to authenticate.
 
-### OAuth 2.0 grant types
+### Grant types
 
-Clerk OAuth 2.0 provider supports the following flows and grant types:
+Clerk's OAuth 2.0 IdP implementation supports the following authorization flows and grant types:
 
 - [Authorization Code](https://oauth.net/2/grant-types/authorization-code/)
 - [PKCE](https://oauth.net/2/pkce/)
 - [Refresh token](https://oauth.net/2/grant-types/refresh-token/)
 
-## Frequently asked questions (FAQ)
+## OpenID Connect (OIDC)
 
-### When do the tokens expire?
+After a user successfully authenticates using the OIDC flow, they receive an ID token along with other tokens.
+
+The ID token is a JWT (JSON Web Token) that contains standard JWT claims as defined in RFC 7519, as well as additional custom claims that represent the authenticated user's profile information. The token is signed using your instance's private key and can be verified using the corresponding public key.
+
+{/* TODO: How do they use the public key to validate the token? */}
```

Comment:
They can validate it as any other regular JWT that has been generated and signed by their instance. For example the Clerk session token. You can obtain the instance public key from the JWKS endpoint. I believe we don't need to add any additional information on how to do it. Maybe a link to a page that describes how to verify their instance JWT is enough